js/popup.js

```diff
@@ -0,0 +1,32 @@
+const languagesAPI = "https://translation.googleapis.com/language/translate/v2/languages?key=${key}";
+const languagesJSON = "../data/languages.json";
+
+document.addEventListener("DOMContentLoaded", () => {
+    loadSupportedLanguages();
+});
+
+/**
+ * Loads list of supported languages into a dropdown.
+ */
+function loadSupportedLanguages() {
+    fetch(languagesJSON)
+        .then(response => response.json()) 
+        .then(data => {
+            const dropdown = document.getElementById("languageDropdown");
+            const userLanguageCode = navigator.language;
+            const userLanguageOption = document.createElement('option');
+            userLanguageOption.value = userLanguageCode;
+            const userLanguage = data.find((language) => language.code === userLanguageCode);
+            userLanguageOption.text = userLanguage.name;
+            dropdown.appendChild(userLanguageOption);
+            data.forEach((language) => {
```

Comment:
i think one thing we should agree on is are we going to use forEach? or more classic for loops? im team classic for loops simply because you end up with less brackets + more readable 

just noticed i mix the two in content.js, i can fix in another pr